packages/typography/src/Typography.tsx

```diff
@@ -0,0 +1,144 @@
+import { uiColors } from '@leafygreen-ui/palette';
+import omit from 'lodash/omit';
+
+const sharedStyles = css`
+  display: block;
+  margin: unset;
+  font-family: Akzidenz, 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  color: ${uiColors.gray.dark2};
+`;
+
+const typeScale1 = css`
+  display: inline-block;
+  font-size: 14px;
+  line-height: 20px;
+  letter-spacing: 0px;
+`;
+
+const typeScale2 = css`
+  display: inline-block;
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 0px;
+`;
+
+const h1 = css`
+  font-weight: 500;
+  font-size: 60px;
+  line-height: 68px;
+  letter-spacing: -0.3px;
+`;
+
+type H1Props = HTMLElementProps<'h1'>;
+
+function H1({ children, className, ...rest }: H1Props) {
+  return (
+    <h1 {...rest} className={cx(sharedStyles, h1, className)}>
+      {children}
+    </h1>
+  );
+}
+
+const h2 = css`
+  font-size: 32px;
+  line-height: 40px;
+  letter-spacing: 0px;
+`;
+
+type H2Props = HTMLElementProps<'h2'>;
+
+function H2({ children, className, ...rest }: H2Props) {
+  return (
+    <h2 {...rest} className={cx(sharedStyles, h2, className)}>
+      {children}
+    </h2>
+  );
+}
+
+const subtitle = css`
+  font-size: 18px;
+  line-height: 24px;
+  letter-spacing: 0px;
+`;
+
+type SubtitleProps = HTMLElementProps<'h6'>;
+
+function Subtitle({ children, className }: SubtitleProps) {
+  return <h6 className={cx(sharedStyles, subtitle, className)}>{children}</h6>;
+}
+
+type BodyProps = HTMLElementProps<'p'> & {
+  /**
+   * font-weight applied to typography element
+   * default: `regular`
+   */
+  weight?: 'regular' | 'medium';
+};
+
+function Body({ children, className, weight = 'regular' }: BodyProps) {
+  const size = useBaseFontSize();
+  const body = size === 16 ? typeScale2 : typeScale1;
+
+  const fontWeight = css`
+    font-weight: ${weight !== 'regular' ? 600 : 400};
+  `;
+
+  return (
+    <p className={cx(sharedStyles, body, fontWeight, className)}>{children}</p>
+  );
+}
+
+const code = css`
+  font-family: 'Source Code Pro', monospace;
+`;
+
+type InlineCodeProps = HTMLElementProps<'code'>;
+
+function InlineCode({ children, className }: InlineCodeProps) {
+  const size = useBaseFontSize();
+  const body = size === 16 ? typeScale2 : typeScale1;
+
+  return (
+    <code className={cx(sharedStyles, code, body, className)}>{children}</code>
+  );
+}
+
+const disclaimer = css`
+  font-size: 12px;
+  line-height: 20px;
+  letter-spacing: 0.2px;
+`;
+
+type DisclaimerProps = HTMLElementProps<'small'>;
+
+function Disclaimer({ children, className }: DisclaimerProps) {
+  return (
+    <small className={cx(sharedStyles, disclaimer, className)}>
+      {children}
+    </small>
+  );
+}
+
+const overline = css`
+  font-size: 12px;
+  font-weight: 600;
+  text-transform: uppercase;
+  line-height: 16px;
+  letter-spacing: 0.4px;
+`;
+
+type OverlineProps<T> = T & BoxProps<T>;
+
+function Overline<T extends React.ReactNode>(props: OverlineProps<T>) {
```

Comment:
What is Overline?

Reply:
<small>**IT'S TEXT LIKE THIS**</small>
<h2>Over something else, like a header</h2>